### PR TITLE
Add destroy function, allowing send funds to recepient.

### LINKED ISSUE #198: Allow Destructible to receive an optional address to which to send funds
Instead of sending always to `owner`

### PR DESCRIPTION
**Pull request by misteraverin**
Closes #198.

### CODE REVIEW

**Comment by misteraverin** [comment]
@maraoz also i run `npm test` several times and after third time it shows me several errors not connecting with my contract.

File: contracts/lifecycle/Destructible.sol [hunk 1]
```diff
@@ -7,9 +7,14 @@ import "../ownership/Ownable.sol";
 /*
  * Destructible
  * Base contract that can be destroyed by owner. All funds in contract will be sent to the owner.
+ * In second function all funds will be sent to the recepient.
```
Comment (maraoz) [inline contracts/lifecycle/Destructible.sol]: I would remove this extra comment you added

File: contracts/lifecycle/Destructible.sol [hunk 2]
```diff
@@ -7,9 +7,14 @@ import "../ownership/Ownable.sol";
...
  */
 contract Destructible is Ownable {
   function destroy() onlyOwner {
     selfdestruct(owner);
   }
+
+  function destroyAndSendRecepient(address _recipient) onlyOwner {
+    selfdestruct(_recipient);
+  }
```
Comment (maraoz) [inline contracts/lifecycle/Destructible.sol]: I don't like the name `destroyAndSendRecepient`. Maybe `destroyAndSendTo` or just `destroyAndSend`

**Comment by maraoz** [comment]
Thanks for the contribution @misteraverin !

File: contracts/lifecycle/Destructible.sol (hunk 2 above)
Comment (misteraverin) [inline contracts/lifecycle/Destructible.sol]: @maraoz actually, I agree. I also couldn't come to satisfation with name. `destroyAndSendTo` sounds good

**Comment by misteraverin** [comment]
@maraoz done!

**Comment by misteraverin** [comment]
@maraoz m?

**Pull request merged**